integrationTests/src/test/java/org/sonarqube/gradle/GradleTest.java

```diff
@@ -227,4 +226,24 @@ public void testJaCoCoProperties() throws Exception {
     assertThat(props.getProperty("sonar.jacoco.reportPaths")).contains(baseDir.resolve("build/jacoco/test.exec").toString());
     assertThat(props.getProperty("sonar.coverage.jacoco.xmlReportPaths")).contains(baseDir.resolve("build/reports/jacoco/test/jacocoTestReport.xml").toString());
   }
+
+  @Test
+  public void testProjectWithConfigurationCacheDoubleExecutionsShouldWork() throws Exception {
+    //--configuration cache was brought by release 6.5.0
+    if (getGradleVersion().isGreaterThan("6.5.0")) {
```

Comment:
wouldn't it be better to use junit's `Assume.assumeTrue`? That way it shows as skipped.

Reply:
Didn't know about that, that's a good one :)